package.json

```diff
@@ -1,23 +1,28 @@
 {
-  "name": "@chanzuckerberg/czedi-kit",
-  "version": "0.0.0",
+  "name": "edu-design-system",
+  "private": true,
   "description": "Learning Platform Design System",
   "main": "index.js",
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "build": "NODE_ENV=production lerna run build",
+    "build": "NODE_ENV=production yarn workspaces foreach run build",
+    "create-component": "plop",
     "create-releases": "lerna version --no-push --conventional-commits --message 'chore(release): %s'",
-    "lint": "npm run lint:styles && npm run lint:scripts",
-    "lint:fix": "npm run lint:styles:fix && npm run lint:scripts:fix",
+    "create-releases:alpha": "yarn create-releases --preid alpha prerelease",
+    "lint": "yarn run lint:styles && yarn run lint:scripts",
+    "lint:fix": "yarn run lint:styles:fix && yarn run lint:scripts:fix",
     "lint:styles": "stylelint --ignore-path .gitignore packages/**/*.{css,js,jsx,ts,tsx}",
-    "lint:styles:fix": "npm run lint:styles -- --fix",
+    "lint:styles:fix": "yarn run lint:styles --fix",
     "lint:scripts": "eslint --ignore-path .gitignore --ext=js,jsx,ts,tsx .",
-    "lint:scripts:fix": "npm run lint:scripts -- --fix",
-    "postinstall": "lerna bootstrap",
+    "lint:scripts:fix": "yarn run lint:scripts --fix",
     "prettier": "prettier --write \"**/*.{js,jsx,ts,tsx}\"",
     "publish-releases": "lerna publish from-git --no-push",
-    "start": "lerna run build --ignore=@chanzuckerberg/eds-components && lerna run start --parallel",
-    "test": "lerna run test",
-    "types": "lerna run types"
+    "publish-releases:alpha": "yarn publish-releases --dist-tag alpha",
```

Comment:
Added scripts for publishing alpha versions.

Normally with Lerna you'd use one command to do this - `lerna publish --canary`.

But we have it split into the two separate phases (create-releases/version and publish-releases/publish), which helps us verify everything is correct before pushing.

That works great for us (and I like it), but it does mean we'll want to make sure each phase's command is correct. Hopefully these scripts help us do that.

Reply:
thanks!